assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -17,21 +17,23 @@
 /**
  * External dependencies
  */
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { rgba } from 'polished';
 
 /**
  * Internal dependencies
  */
-import {
-  TypographyPresets,
-  StandardViewContentGutter,
-  Button,
-  TextInput,
-} from '../../../components';
+import { StandardViewContentGutter } from '../../../components';
 import { visuallyHiddenStyles } from '../../../utils/visuallyHiddenStyles';
-import { Link } from '../../../components/link';
-import { BUTTON_TYPES, KEYBOARD_USER_SELECTOR } from '../../../constants';
+import { KEYBOARD_USER_SELECTOR } from '../../../constants';
```

Comment:
Anyway to remove the need for this user selector now that we're using the new keyboard focus style?